### PR TITLE
bug 1557864 incorrect description of cli wrapper

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

**Comment by bfallonf** [comment]
@kalexand-rh LGTM :+1:

File: install_config/install/rpm_vs_containerized.adoc [hunk 1]
```diff
@@ -145,27 +144,17 @@ also provided to ease administrative tasks:
...
-The wrapper scripts mount a limited subset of paths:
-
-- *_~/.kube_*
-- *_/etc/origin/_*
-- *_/tmp/_*
-
-Be mindful of this when passing in files to be processed by the `oc` or `oadm`
-commands. You may find it easier to redirect the input, for example:
+Be mindful of these links when you pass in files to be processed by the `oc` or
+`oadm` commands. You might find it easier to redirect the input, for example:
```
Comment (vikram-redhat) [inline install_config/install/rpm_vs_containerized.adoc]: We have started deprecating the use of oadm command. So might be good opportunity to remove it here.

**Comment by vikram-redhat** [comment]
Great work here @kalexand-rh! Thanks! @bfallonf if you are done with the peer review, you need to remove the peer review label and put the done label on.

**Comment by bfallonf** [comment]
@vikram-redhat Oops

**Comment by kalexand-rh** [comment]
CP for 3.9: https://github.com/openshift/openshift-docs/pull/8257

**Pull request merged**